docs/plugins/plugin-script.asciidoc

```diff
@@ -13,6 +13,13 @@ Run the following command to get usage instructions:
 sudo bin/plugin -h
 -----------------------------------
 
+[float]
```

Comment:
Perhaps make this an IMPORTANT block, eg:

     [IMPORTANT]
    .Running as root
    ===========================
    If Elasticsearch was installed using the deb or rpm package then run
    `bin/plugin` as `root` so it write the appropriate files on disk. Otherwise run
    `bin/plugin` as the user that owns all of the Elasticsearch files.
    ===========================